src/js/events/MetronomeActions.js

```diff
@@ -74,10 +65,21 @@ const MetronomeActions = {
               });
               resolve();
             } catch (error) {
-              handleError(error);
+              AppDispatcher.handleServerAction({
+                type: REQUEST_METRONOME_JOBS_ERROR,
+                data: error
+              });
```

Comment:
I think the consumer should always check if `xhr == null` first. That would be the safest option.